--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33 +33 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
--- a/setup.py
+++ b/setup.py
@@ -18 +18 @@
-            subprocess.check_call(['cmake', '../..', '-DCMAKE_CXX_FLAGS=-fPIC'],
+            subprocess.check_call(['cmake', '../..', '-DCMAKE_CXX_FLAGS=-fPIC', '-DCMAKE_OSX_DEPLOYMENT_TARGET=26.0'],
--- a/setup.py
+++ b/setup.py
@@ -30 +30 @@
-              extra_compile_args=['-fPIC']),
+              extra_compile_args=['-fPIC', '-mmacosx-version-min=26.0'], extra_link_args=['-mmacosx-version-min=26.0']),
--- a/setup.py
+++ b/setup.py
@@ -34,0 +35 @@
+    version="0.0.0",
